client/utils/Table.cc

```diff
@@ -111,15 +105,25 @@ void TableRow::dumpTo (ostream &stream, const Table & parent) const
     }
     else
     {
-      stream << s;
-      stream.width (parent._max_width[c] - ssize);
+      if (parent._header.align(c) == TableAlign::LEFT)
+	stream << s << setw(parent._max_width[c] - ssize) << "";
+      else
+	stream << setw(parent._max_width[c] - ssize) << "" << s;
```

Comment:
I am not sure if it is bug in github, but indentation looks strange here.

Reply:
Somehow github does not handle the spaces and tabs correct.

Reply:
FYI, GitHub assumes the Tab character has stops each 4 columns while the text terminals assume 8 columns. So it's best not to mix tabs and spaces in files.